src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.const.ts

```diff
@@ -0,0 +1,14 @@
+import i18n from 'i18n';
+
+const { t } = i18n;
+
+export const SLIDER_VALUE_HEAD_ROWS = [
```

Comment:
did you check if it's translate or not?
because usually we use functions to get translations, for ex. getSliderValueRows = () => [...]

Reply:
done